backend/src/main/scala/bloop/BloopClassFileManager.scala

```diff
@@ -41,7 +41,7 @@ final class BloopClassFileManager(
   private[this] val generatedFiles = new mutable.HashSet[File]
 
   // Supported compile products by the class file manager
-  private[this] val supportedCompileProducts = List(".sjsir", ".nir", ".tasty")
+  val supportedCompileProducts = List(".sjsir", ".nir", ".tasty", ".betasty")
```

Comment:
Could we just put it into an object? We wouldn't need to pass manager in the params